File: uri/C/beginner/1064.c
#include <stdio.h>

int main()
{

	double X, Y, Z, T, Q, W, media;
	int i;

	scanf("%lf %lf %lf %lf %lf %lf", &X, &Y, &Z, &T, &Q, &W);

	if(X > 0){
		i += 1;
		media += X;
	}

	if(Y > 0){
		i += 1;
		media += Y;
	}

	if(Z > 0){
		i += 1;
		media += Z;
	}

	if(T > 0){
		i += 1;
		media += T;
		}

	if(Q > 0){
			i += 1;
			media += Q;
		}

	if(W > 0){
			i += 1;
			media += W;
		}

	printf("%d valores positivos\n", i);
	printf("%.1lf\n", media/i);

	return 0;
}

File: uri/python/1064.py
# -*- coding: utf-8 -*-

primeiro = float(input())
segundo = float(input())
terceiro = float(input())
quarto = float(input())
quinto = float(input())
sexto = float(input())

positivos = 0
media = 0

if primeiro > 0:
	positivos = positivos + 1
	media = media + primeiro
if segundo > 0:
	positivos = positivos + 1
	media = media + segundo
if terceiro > 0:
	positivos = positivos + 1
	media = media + terceiro
if quarto > 0:
	positivos = positivos + 1
	media = media + quarto
if quinto > 0:
	positivos = positivos + 1
	media = media + quinto
if sexto > 0:
	positivos = positivos + 1
	media = media + sexto

print('{} valores positivos'.format(positivos))
print('{:.1f}'.format(media/positivos))

